### DISCUSSION TITLE
Trivy error on v1.5

### DISCUSSION THREAD

**Discussion by LtCmdrScott** [rancher-sandbox/rancher-desktop#2702] [General]
I am getting this error in my log when I try to startup Rancher v1.5 on my ARM desktop.  We have to run Rancher without admin rights in our environment, so I need to find a way around this issue without enabling admin rights:

2022-08-08T18:51:42.448Z: > limactl copy /Applications/Rancher Desktop.app/Contents/Resources/resources/linux/internal/trivy 0:./trivy
2022-08-08T18:51:42.467Z: > limactl shell --workdir=. 0 sudo mv ./trivy /usr/local/bin/trivy
$ Error: spawn /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl EACCES
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -13,
  code: 'EACCES',
  syscall: 'spawn /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl',
  path: '/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl',
  spawnargs: [
    'shell',
    '--workdir=.',
    '0',
    'sudo',
    'mv',
    './trivy',
    '/usr/local/bin/trivy'
  ]
}

The above error is most prevalent, but I am also getting this error from time to time:

2022-08-08T18:57:08.365Z: > limactl list --json
$ Error: spawn /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl EACCES
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -13,
  code: 'EACCES',
  syscall: 'spawn /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl',
  path: '/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl',
  spawnargs: [ 'list', '--json' 
[lima.log](https://github.com/rancher-sandbox/rancher-desktop/files/9284477/lima.log)
]

**Comment by LtCmdrScott** [comment]
Solved the issue!

**Reply by soundweaverz** [reply]
I'm having the same issue! How did you solve it?

**Reply by LtCmdrScott** [reply]
We prepackage up all of our software to deliver to end users (they are NOT allowed to have admin rights).  I delivered only the app and the Kubernetes files (located inside ~/Library/Caches/Rancher-Desktop/.  I made the mistake of also delivering all of the files in the ~/Library/application support/rancher-desktop & Rancher Desktop folders as well...that was my mistake.  I needed to let rancher build those files in Application Support vs me trying to deliver them.

**Reply by soundweaverz** [reply]
Perfect, thank you! 😄
